src/main/java/com/amazon/opendistroforelasticsearch/security/filter/OpenDistroSecurityFilter.java

```diff
@@ -355,6 +356,9 @@ public void onFailure(Exception e) {
                 listener.onFailure(new ElasticsearchSecurityException(err, RestStatus.FORBIDDEN));
                 return;
             }
+        } catch (ElasticsearchException err) {
+            log.error("Unexpected exception "+ err, err);
```

Comment:
This is an anti-pattern. There is no point to log exception message twice.

Reply:
I followed the existing code style of [this line](https://github.com/opendistro-for-elasticsearch/security/blob/f3e474c1783e7afc592dcab4f6b72806abc47281/src/main/java/com/amazon/opendistroforelasticsearch/security/filter/OpenDistroSecurityFilter.java#L370) Does this look better now?

Reply:
When logging an exception, it is always better to have stack trace logged as well. It is also necessary to avoid premature string concatenation and rely on place holder instead. I don't think that "Unexpected" is the right message here. It is expected for the wrong parameters provided (wrong index name). I also think that logging it at error level is not necessary and debug (or warning) should be sufficient.

Reply:
Changed the exception logs.